packs/icinga2/sensors/lib/client.py

```diff
@@ -0,0 +1,136 @@
+import pycurl, json
+import urllib
+import signal
+import sys
+import time
+
+POST_PARAMS = {}
+
+class Client:
```

Comment:
I see :)

If you need to poll you can achieve same thing with a while loop (or similar) + some requests code.